src/js/requestHandler.js

```diff
@@ -0,0 +1,33 @@
+const requestHandler = {
+    
+    initRequest(request, component) {
```

Comment:
from a js standpoint this functionally works however does it need to be tied to a "component"?